content/en/blog/2021/istio-4th-birthday/index.md

```diff
@@ -0,0 +1,65 @@
+- We simplified debugging experience with various istioctl commands.
+- We expanded the mesh to services running in [VMs](https://istio.io/latest/news/releases/1.9.x/announcing-1.9/#virtual-machine-integration-beta)
+    and [multiclusters](https://istio.io/latest/docs/setup/install/multicluster/).
+- We made [statefulset easier](https://istio.io/latest/blog/2021/statefulsets-made-easier/) to use in Istio 1.10 with zero-configuration.
+- We made various performance improvements to Istio control plane and data plane via [discovery selectors](https://istio.io/latest/blog/2021/discovery-selectors/),
+    sidecar resources etc.
+- We [introduced WebAssembly](https://istio.io/latest/blog/2021/wasm-progress/) as our extensibility platform which has helped users tailor Istio to their needs
+- We beefed up our CVE management and release processes to meet enterprise needs.
+
+[Read more](https://istio.io/latest/blog/2020/tradewinds-2020/) about improvements to
+Istio in 2020 that made this technology easier to use.
+
+In February 2021, we celebrated the first IstioCon! This community-led event was an opportunity
+for users and developers to share [many examples](https://www.youtube.com/playlist?list=PL7wB27eZmdffS-g_xh7X-b0echc_XZMKV)
+of how they use Istio in production and lessons
+learned from it. IstioCon was a great opportunity to feature more than 25 end-user companies,
+like Salesforce, T-Mobile, and Airbnb, among others, to feature maintainers from across the Istio
+ecosystem, and to share the [Istio project roadmap](https://www.youtube.com/watch?v=WmjTeN-jtdY).
+
+This inaugural community conference was a major success, with more than 4,000 registrants from 80
+countries that participated in this conference. The program was conducted on US and Asia time zones,
+and on English and Chinese languages to accommodate big user communities in various continents.
+Learn more about the [impact](https://events.istio.io/istiocon-2021/slides/IstioCon2021-Report.pdf)
+of IstioCon and find the [presentations](https://events.istio.io/istiocon-2021/sessions/) on
+the conference website.
+
+## How to participate in Istio’s 4th Birthday celebration
+
+Contributions are key to the long life of an open source project. This is why, on its 4th birthday,
+we want to hear about your contributions to the Istio project. To participate in this campaign,
+share on Twitter a contribution you made to the project and why it matters, using the hashtag
+#IstioTurns4 and #IstioBirthday. You can submit posts from Monday, May 24th at 9 am PT, until
+Friday, May 28th, at 12pm PT, to enter a chance to win some Istio swag.
+
+The other way of participating in this campaign is by joining the Isito community meetup, which
+will take place on Thursday, May 27th at 10 am PT. At this event, we will have Pratima Nambiar
+present about contributions that stem from using Isito in production at Salesforce. Join the event,
```

Comment:
s/present about/discuss.  Not sure how to change this.

Reply:
fixed. WDYT?